Libraries/HyperV.psm1

```diff
@@ -303,6 +303,17 @@ function Create-HyperVGroupDeployment([string]$HyperVGroupName, $HyperVGroupXML,
                     continue
                 }
             }
+            # Check host memory before creating any VM
+            if ($CurrentTestData.OverrideVMSize) {
+                $CurrentVMSize = $CurrentTestData.OverrideVMSize
+            } else {
+                $CurrentVMSize = $VirtualMachine.ARMInstanceSize
+            }
+            Set-Variable -Name HyperVInstanceSize -Value $CurrentVMSize -Scope Global
+            $CurrentVMCpu = $HyperVMappedSizes.HyperV.$CurrentVMSize.NumberOfCores
+            $CurrentVMMemory = [int]$HyperVMappedSizes.HyperV.$CurrentVMSize.MemoryInMB
+            Get-HostMemory -HvServer $HyperVHost -RequestedMemory $CurrentVMMemory
+            $CurrentVMMemory = [int]$CurrentVMMemory * 1024 * 1024
```

Comment:
you can do * 1GB, it will work out of the box

Reply:
* 1MB, but yes, good catch